docs/source/rules.rst

```diff
@@ -102,10 +104,31 @@ In the above example, we are evaluating the ``pci`` matcher.  As you can likely
 outputs
 ~~~~~~~
 
-``outputs`` define where the alert should be sent to, if the return value of the function is ``True``.
+``outputs`` define where the alert should be sent to, if the return value of the rule is ``True``.
```

Comment:
you capitalize this on L35, but downcase it here. i think you might want to choose either one or the other for consistency